.yamato/package-test.yml

```diff
@@ -0,0 +1,136 @@
+{% metadata_file .yamato/package.metafile -%}
+---
+
+{% for editor in test_editors %}
+{% for platform in test_platforms %}
+test_{{ platform.name }}_{{ editor.version }}:
+  name : Test {{ editor.version }} on {{ platform.name }}
+  agent:
+    type: {{ platform.type }}
+    image: {{ platform.image }}
+    flavor: {{ platform.flavor}}
+  variables:
+    UPMCI_PKG: "upm-ci-utils@stable"
+  commands:
+      - npm install {% if platform.name == "win" %}"%UPMCI_PKG%"{% else %}"$UPMCI_PKG"{% endif %} -g --registry {{ registry.npm }}
+      - upm-ci package test -u {{ editor.version }} --package-path package/com.unity.formats.usd --extra-create-project-arg=-upmNoDefaultPackages --extra-utr-arg="--api-profile=NET_4_6"
+  artifacts:
+    logs:
+      paths:
+        - "upm-ci~/test-results/**/*"
+        - "upm-ci~/upm-ci.log"
+  dependencies:
+    - .yamato/package-pack.yml#pack
+{% endfor %}
+{% endfor %}
+
+{% for editor in test_editors %}
+{% for platform in test_platforms %}
+test_recorder_{{ platform.name }}_{{ editor.version }}:
+  name : Test Recorder {{ editor.version }} on {{ platform.name }}
+  agent:
+    type: {{ platform.type }}
+    image: {{ platform.image }}
+    flavor: {{ platform.flavor}}
+  variables:
+    UPMCI_PKG: "upm-ci-utils@stable"
+  commands:
+      - npm install {% if platform.name == "win" %}"%UPMCI_PKG%"{% else %}"$UPMCI_PKG"{% endif %} -g --registry {{ registry.npm }}
+      - upm-ci project test  --project-path TestProject/Usd-Recorder --type project-tests --unity-version {{ editor.version }} --extra-utr-arg="--api-profile=NET_4_6"
```

Comment:
Should this have --extra-create-project-arg=-upmNoDefaultPackages?

Reply:
It might also be good to reorder the arguments and use -u {{ editor.version }} to keep this consistent with the package tests

Reply:
Hmm, I'm not really sure if it needs the --extra-create-project-arg=-upmNoDefaultPackages 🤔 It's using the Test Project Usd-Recorder, so I wouldn't expect it to be creating a new Unity project to test in, but the documentation still calls it out 😅 I'll add it in and if the tests still all pass we can go with it